src/lev-report/charts/HourlyUsage.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { VictoryAxis, VictoryChart, VictoryLabel, VictoryLine, VictoryTheme } from 'victory';
+
+const hours = Array.from({ length: 25 }, (_, h) => h);
+
+const HourlyUsage = ({ traces }) => traces && traces.length &&
+<VictoryChart
+	padding={{ top: 10, bottom: 50, left: 50, right: 5 }}
+	domainPadding={20}
+	theme={VictoryTheme.material}
+	style={{ labels: { fontSize: 10 } }}
+	height={250}
+	width={300}
+>
+	<VictoryAxis
+		domain={[0, 24]}
+		tickValues={[0, ...hours.filter(n => !(n % 3))]}
+		tickFormat={t => t < 10 ? '0' + t : t}
```

Comment:
Is there not a 'formatting' function we could use here? I think that would make the intention more obvious.

Reply:
There is not a 'formatting' function... other than the one you see here in the code

Reply:
I suppose I was hoping for some sort of `sprintf` for javascript. There are some libraries but nothing in the language itself. :-(